app/src/main/java/org/mozilla/fenix/compose/button/TextButton.kt

```diff
@@ -20,23 +24,36 @@ import java.util.Locale
  * Text-only button.
  *
  * @param text The button text to be displayed.
+ * @param shouldUppercaseText Whether or not the text should be uppercase.
+ * @param buttonColors [ButtonColors] for the button.
  * @param onClick Invoked when the user clicks on the button.
  * @param modifier [Modifier] Used to shape and position the underlying [androidx.compose.material.TextButton].
+ * @param contentPaddingValues Padding values for the button's content
  * @param textColor [Color] to apply to the button text.
  */
 @Composable
+@Suppress("LongParameterList")
 fun TextButton(
     text: String,
+    shouldUppercaseText: Boolean = true,
+    buttonColors: ButtonColors = textButtonColors(),
```

Comment:
why was `buttonColors` added as a parameter? from what I can tell, it's only being used for a text color, which is already covered by the `textColor` parameter

Reply:
Gotcha gotcha. If you haven't already, could you double check with UX what the ripple color should be? I'm unable to find a token for this with our current color system.